lab1.js

```diff
@@ -117,18 +142,24 @@ var tooHungryDay;
  pondering protein supplements (the first day the average dips below 4
  meals)
 */
+var tooHungryDay = 0;
 
-// your code goes here
+for ( var i = 0; i < mealsPerDay.length; i++ ) {
+	console.log( mealsPerDay[i] / i );
+	if ( mealsPerDay[i] < 4 ) {
```

Comment:
Close. you want to return the first day that the average of meals drops below 4. Here, your looking at whether the meals for that day drop below 4. You'll probably want some sort of "average" variable to keep track of that.